### PR TITLE
python-certifi 2023.7.22 (new formula)

### PR DESCRIPTION
**Pull request by alebcay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

From https://github.com/orgs/Homebrew/discussions/4691.

Couple of notes:
- The versioning for this formula is a bit moot, seeing as the version of the certificates is now based on `ca-certificates` formula
- Is a caveats needed? Left it out for now
- Not sure whether to name as `certifi` or `python-certifi` (we have `six` and `python-tabulate`)

### CODE REVIEW

**Comment by chenrui333** [comment]
> * Not sure whether to name as `certifi` or `python-certifi` (we have `six` and `python-tabulate`)

I think `six` should be really as `python-six`

File: Formula/python-certifi.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      site_packages = Language::Python.site_packages("python#{python.version.major_minor}")
+      rm prefix/site_packages/"certifi/cacert.pem"
+      (prefix/site_packages/"certifi").install_symlink Formula["ca-certificates"].pkgetc/"cert.pem" => "cacert.pem"
+    end
+  end
+
+  test do
+    pythons.each do |python|
+      python_exe = python.opt_libexec/"bin/python"
+      output = shell_output("#{python_exe} -c 'import certifi; print(certifi.where())'").chomp
```
Comment (carlocab) [inline Formula/python-certifi.rb:32]:
```suggestion
      output = shell_output("#{python_exe} -m certifi").chomp
```
I think this is equivalent

**Review by chenrui333 — APPROVED** [review]
LGTM

**Comment by alex** [comment]
Once this is merged, what's the process for migrating each (relevant) formula to use it? Just send PRs as we have use cases?

**Comment by alebcay** [comment]
> Once this is merged, what's the process for migrating each (relevant) formula to use it? Just send PRs as we have use cases?

I would think so. I don't foresee any need to coordinate formula updates to use this.

**Pull request merged**